src/Components/Radio/Radio.component.jsx

```diff
@@ -77,14 +78,29 @@ const Radio = ({activeTrack, setActiveTrack}) => {
   const [status, setStatus] = useState(0);
   const [audio, setAudio] = useState(null);
   const [playing, setPlaying] = useState(false);
-  let nowPlaying = '<Song Name>';
-
-  if (activeTrack) {
-    nowPlaying = activeTrack.title;
-  }
+  //let nowPlaying = '<Song Name>';
+  const [nowPlaying, setNowPlaying] = useState('<CLICK PLAY!>');
```

Comment:
nit (nit pick, no big deal)

nice, maybe it would be cleaner if click play was in the status bank?

Reply:
perfect. this will help it read like a state diagram